### PR TITLE
Emergency hotfix for engi crossbow [NO GBP]

### PR DESCRIPTION
**Pull request by KingkumaArt**
Hadnt added these commits when the PR got merged due to not expecting it oops

## Why It's Good For The Game

See above, used to make my last pr actually function as intended.

## Changelog

:cl:
fix: fixed engi crossbow being able to be used onehanded + ability to craft with sci inducers
/:cl:

### CODE REVIEW

**Comment by Jacquerel** [comment]
Please draft your PRs when you don't want them to be merged :p

File: code/datums/components/crafting/ranged_weapon.dm [hunk 1]
```diff
@@ -214,6 +214,9 @@
 		/obj/item/stack/cable_coil = 12,
 		/obj/item/inducer =  1,
 	)
+	blacklist = list(
+	/obj/item/inducer/sci,
```
Comment (Jacquerel) [inline code/datums/components/crafting/ranged_weapon.dm:218]:
```suggestion
		/obj/item/inducer/sci,
```

**Review by Jacquerel — CHANGES_REQUESTED** [review]

**Comment by KingkumaArt** [comment]
> Please draft your PRs when you don't want them to be merged :p

Sorry I didnt think it could be merged as I hadnt heard back from melbert

**Comment by deathrobotpunch** [comment]
also the xbow and it’s ammo isn’t craftable as far as I know (checked code and crafting menu)

**Comment by KingkumaArt** [comment]
> also the xbow and it’s ammo isn’t craftable as far as I know (checked code and crafting menu)

yeah ive noticed this, and im thinking its a merge weirdness thing

**Review by Ghommie — APPROVED** [review]

**Pull request merged**